src/main/java/ch/naviqore/gtfs/schedule/model/GtfsScheduleBuilder.java

```diff
@@ -111,6 +109,10 @@ public GtfsScheduleBuilder addCalendarDate(String calendarId, LocalDate date, Ex
     }
 
     public GtfsScheduleBuilder addTrip(String id, String routeId, String serviceId, String headSign) {
+        return addTrip(id, routeId, serviceId, headSign, TripAccessibilityType.UNKNOWN, TripBikeInformation.UNKNOWN);
+    }
+
+    public GtfsScheduleBuilder addTrip(String id, String routeId, String serviceId, String headSign, TripAccessibilityType wheelchairAccessible, TripBikeInformation bikesAllowed) {
```

Comment:
Maybe name the variable the same as the type? `accessibilityType` and `bikeInformation`.

Reply:
I would keep naming consistent to the column names in the gtfs text files that's why I wouldn't adjust.

Reply:
Good point.